frontend/src/components/explorer/mapViewer/MapControls.vue

```diff
@@ -121,7 +117,3 @@ export default {
 };
 
 </script>
-
-<style lang="scss">
-
-</style>
```

Comment:
This is a bit weird to remove IMO, we have this in many of the components and is beneficial to keep for future styling

Reply:
Thanks for noticing this. Apparently we had an inconsistent setup in our code base, now addressed by 465fefab.